xunit.runner.visualstudio.testadapter/VsTestRunner.cs

```diff
@@ -307,9 +309,13 @@ static bool IsXunitTestAssembly(string assemblyFileName)
             if (platformAssemblies.Contains(Path.GetFileName(assemblyFileName)))
                 return false;
 
+#if DOTNET_CORE
+            return true;
```

Comment:
One way is to read the *.deps.json file to figure out whether xunit or xunit.extensibility.execution libraries are added or not. But then, we need to load *.deps.json file and parse it.